bin/get-valid-instance-types.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+for instance in $(echo "$(aws ec2 describe-spot-price-history help)" | grep -e "^[\t ]*[a-z][0-9]\." | tr -d ' '); do
```

Comment:
Is this just to enumerate it for you?

Reply:
yeah, this just goes through each one and prints it to make copy paste as easy as possible.

I could automate updating all the files but didn't want to spend any time on it so decided this was enough automation.